subworkflows/local/bam_variant_calling_cnvkit/main.nf

```diff
@@ -21,6 +23,15 @@ workflow BAM_VARIANT_CALLING_CNVKIT {
 
     CNVKIT_BATCH(cram, fasta, fasta_fai, targets, reference, generate_pon)
 
+    // right now we do not use an input VCF to improve the calling of B alleles
+    // based on SNV frequencies from the VCF file
+    // in the future we might consider to add this, by connecting the emission from
+    // SNV variant calling modules
+    CNVKIT_CALL(CNVKIT_BATCH.out.cns.map{ meta, cns -> [meta, cns[2], []]})
+
+    // export to VCF for compatibility with other tools
+    CNVKIT_EXPORT(CNVKIT_CALL.out.cns)
```

Comment:
Why isn't the vcf-output from `CNVKIT_EXPORT` emitted from this subworkflow? I think in most cases, variant-calling subworkflow emitted vcf-files so that the can be annotated (and what not) downstream.

Reply:
That's a very valid point

Reply:
absolutely valid point, it wasn't intentional :P